src/components/author/author.twig

```diff
@@ -1,23 +1,76 @@
+{#
+  This component accepts the following properties
+  - `meta` (optional, string)
+  - `date` (optional, date object)
+
+  If both the `meta` and `date` properties contain values, the `meta` value has
+  a higher priority and will override the `date` value.
+
+  It is recommened to use the `date` property when wanted to render a date as the
+  meta data so that a more a accessible user experience can be rendered.
+#}
+{% set _meta_content %}
+  {% block meta_content %}
+    {% if meta %}
+      <p>{{ meta }}</p>
+    {% elseif date %}
+      <p>
+        {# This creates a more accessible, and less confusing, user experience #}
+        <span class="u-hidden-visually">
+          Published on {{ date|date('F jS, Y') }}
```

Comment:
The "Published on" text should probably be a template property so it can be changed if needed.

Reply:
Yeah, that's a good idea now that the use cases have been broadened. Also a good way to future-proof. But I think this is a great default value!

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1607/commits/b6e22592b0fd39ad91ead1037f0c1671be00b4d5